### ISSUE TITLE
sample code in recipes/network.rb comment results in a recipe compile error

### ISSUE THREAD

**Issue by cmessinaclearchannel** [DataDog/chef-datadog#423]
Using the sample code in recipes/network.rb, chef will give a recipe compile error.
https://github.com/DataDog/chef-datadog/blob/master/recipes/network.rb#L5
           
`# node.datadog.network.instances = [`

I believe that this should read: 
`# node.override['datadog']['network']['instances'] = [`

This error occurs on Chef 12.12.15, and presumably 13+ though I have not tested that. 

```
================================================================================
  Recipe Compile Error in /var/chef/cache/cookbooks/blah/recipes/datadog-confs.rb
  ================================================================================
  
  Chef::Exceptions::ImmutableAttributeModification
  ------------------------------------------------
  Node attributes are read-only when you do not specify which precedence level to set. To set an attribute use code like `node.default["key"] = "value"'
  
  Cookbook Trace:
  ---------------
    /var/chef/cache/cookbooks/blah/recipes/datadog-confs.rb:66:in `from_file'
    /var/chef/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/run_context.rb:347:in `load_recipe'
  
  Relevant File Content:
  ----------------------
  /var/chef/cache/cookbooks/blah/recipes/datadog-confs.rb:
  
   65:  #network
   66>> node.datadog.network.instances = [
   67:    {
   68:      :collect_connection_state => "false",
   69:      :excluded_interfaces => ["lo","lo0"]
   70:    },
   71:  ]
   72:  include_recipe "datadog::network"
```

**Comment by olivielpeau** [comment]
Thanks @cmessinaclearchannel! The syntax in these examples is indeed not correct anymore (and we probably use the same syntax in a lot of other recipe comments).

As always, we'd be happy to review a PR of yours, but if you don't have time we'll fix this for the next minor release.

**Comment by olivielpeau** [comment]
Fixed in #434, closing
